tools/runner/runner.go

```diff
@@ -146,9 +146,16 @@ func (r *Runner) runTest(ctx context.Context, config *grpcv1.LoadTest, reporter
 				reporter.Error("Could not save pod logs: %s", err)
 			}
 			reporter.AddProperty("name", loadTest.Name)
-			for property, value := range savedLogs.GenerateProperties(loadTest) {
+			for property, value := range savedLogs.GenerateNameProperties(loadTest) {
```

Comment:
I would leave the "pod." and "log." out of the functions that generate properties, and instead add that here in the loops.

I am actually thinking that we could pass in a []string of prefixes to the property functions and join the properties in a standardized way inside.